### ISSUE TITLE
"say thanks" button not working

### ISSUE THREAD

**Issue by rahilarious** [ogham/exa#1010]
just wanted to thank u for free :)

**Comment by ariasuni** [comment]
I’m not sure where you find this, can you tell me? But I remember finding this somewhere and seeing that the website doesn’t work anymore, yeah.

**Comment by eggbean** [comment]
I think he means on the `README.md`.

![image](https://user-images.githubusercontent.com/815311/158882149-8e65f0e7-b8ae-4342-bd7e-8640fdc60d05.png)

**Comment by ariasuni** [comment]
Yeah the website seems to work, but the link doesn’t. We can probably remove it (and @ogham can re-add it back with a working link later if wanted).

**Comment by cdstanford** [comment]
+1, also wanted to say thanks after installing! @ariasuni Since you said the website does seem to work, do you have the correct link? I can only find [https://saythanks.io/to/ogham%40bsago.me](https://saythanks.io/to/ogham%40bsago.me)

**Comment by ariasuni** [comment]
I tried to find one, but couldn’t. That’s why I said we should probably remove it.

**Comment by ariasuni** [comment]
Closing this since exa has been unmaintained for a while (see https://github.com/ogham/exa/issues/1243), see [the active fork eza](https://github.com/eza-community/eza).
